changelogs/client_server/newsfragments/2074.clarification

```diff
@@ -0,0 +1 @@
+Deduplicate the definitions of `Invite3pid`.
```

Comment:
as per #2073, this should probably be an "internal" change?

Reply:
Moved